mis_builder_contract/models/mis_contract_line.py

```diff
@@ -0,0 +1,69 @@
+from os.path import dirname, join
+
+from odoo import api, fields, models, tools
+
+
+class ContractLineMIS(models.Model):
```

Comment:
Follow naming guidelines, but anyway, check the rest of the comments.

```suggestion
class ContractLine(models.Model):
```